packages/unmock-core/src/__tests__/serviceStore.test.ts

```diff
@@ -63,58 +63,60 @@ describe("Fluent API and Service instantiation tests", () => {
 
   it("Store with existing path does not throw", () => {
     const store = stateFacadeFactory(PetStoreWithPseudoResponses);
-    store.petstore(); // Should pass
+    store.petstore({}); // Should pass
```

Comment:
Hmm, why didn't this throw an error before if `store.petstore()` does not make sense? 🤔

Reply:
I updated it just to fix the type hinting, behind the scenes an empty object is treated the same as undefined object (no-op). Not sure if it should throw.

Reply:
Hmm, I don't understand what there was to "fix" if users can anyway do either 🤔

Reply:
New types mean vscode underlines such mistakes (even though both are acceptable) - 
![image](https://user-images.githubusercontent.com/44643609/61780606-8625ac80-ae0b-11e9-910b-6b6cbc7c88a9.png)

Alternatively ofc I could include 0 arguments as valid, if you feel that's more correct.

Reply:
No, just wondering why `tsc` compiled if it's wrong, maybe we don't even do type checking for the test files.

Reply:
It'll now attempt to compile tests as well.

Reply:
Cool, I'll probably have to test that out at some point as I don't understand how adding `tsconfig.json` to the test folders adds the type checking, unless that's some lerna magic...

Reply:
Ah, some lerna magic. Running `tsc --build tsconfig.test.json` runs `tsc` in all the given `paths` in that file and looks for `tsconfig.json` in each. If it doesn't find it - it crashes.
So of the two options - commenting packages out or adding a boilerplate, I chose the latter :)

Reply:
Cool, do we run that anywhere in CI/CD?

Reply:
Yup, we use `npm ci` in CI, which implicitly calls `postinstall`, which calls `compile` :innocent:

Reply:
Interesting, if compiling test and source code happen in the same step I sure hope lerna hows to keep the build clean!

Reply:
That's taken care of by `tsc --build tsconfig.test.json --clean`, which removes the compiled test files.

Reply:
Ok, I'm still so suspicious that I'll probably have to test adding a type check error in the test files and see what happens myself but this PR is of course very good to go!

Reply:
Okay I just noticed I forgot the update to `package.json`, so that clarifies all this confusion hopefully. :man_facepalming: My bad.